blazor/treegrid/rows/row-height.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: Row Height in Blazor Tree Grid Component | Syncfusion
+description: Checkout and learn here all about Row Height in Syncfusion Blazor Tree Grid component and much more details.
+platform: Blazor
+control: Tree Grid
+documentation: ug
+---
+
+# Row Height in Blazor Tree Grid Component
+
+The row height of tree grid rows can be customized through the [RowHeight](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor~Syncfusion.Blazor.TreeGrid.SfTreeGrid~RowHeight.html) property. The `RowHeight` property is used to change the row height of the entire tree grid rows.

```

Comment:
The row height of tree grid rows can be customized through the [RowHeight](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor~Syncfusion.Blazor.TreeGrid.SfTreeGrid~RowHeight.html) property. The `RowHeight` property changes the row height of the entire tree grid rows.